styleguide.config.js

```diff
@@ -92,7 +92,18 @@ module.exports = {
   template: {
     head: {
       meta: [
-        { name: 'viewport', content: viewportMeta }
+        { name: 'viewport', content: viewportMeta },
+        //Open Graph
+        { property: 'og:title', content: 'vo·cab·u·lar·y' },
+        { property: 'og:type', content: 'website' },
+        { property: 'og:url', content: 'https://creativecommons.github.io/cc-vocabulary/' },
+        { property: 'og:description', content: 'A cohesive design system to unite the web-facing Creative Commons' },
+        { property: 'og:image', content: 'https://creativecommons.github.io/cc-vocabulary/titlecard.png' },
+        // Twitter
+        { name: 'twitter:title', content: 'vo·cab·u·lar·y' },
```

Comment:
Since the values are being repeated in OG and Twitter meta tags, it would be a good idea to define them in variables (such as `viewportMeta` above) and use them in both places.